packages/fx-core/src/component/driver/deploy/azure/azureAppServiceDeployDriver.ts

```diff
@@ -23,68 +18,35 @@ export class AzureAppServiceDeployDriver implements StepDriver {
   readonly description: string = getLocalizedString(
     "driver.deploy.deployToAzureAppServiceDescription"
   );
+  private static readonly SERVICE_NAME = "Azure App Service";
+  private static readonly SUMMARY = [
+    getLocalizedString("driver.deploy.azureAppServiceDeploySummary"),
+  ];
+  private static readonly SUMMARY_PREPARE = [
+    getLocalizedString("driver.deploy.notice.deployDryRunComplete"),
+  ];
+
   @hooks([addStartAndEndTelemetry(ACTION_NAME, TelemetryConstant.DEPLOY_COMPONENT_NAME)])
   async run(args: unknown, context: DriverContext): Promise<Result<Map<string, string>, FxError>> {
-    const impl = new AzureAppServiceDeployDriverImpl(args, context);
-    return wrapRun(
-      () => impl.run(),
-      () => impl.cleanup(),
-      context.logProvider
+    const impl = new AzureZipDeployDriver(
```

Comment:
I am a little confused on why it only call AzureZipDeployDriver.run(), maybe I will check with you offline.

Reply:
class name and file name changed.